context/context.go

```diff
@@ -78,6 +79,8 @@ type Context interface {
 	StmtRollback()
 	// StmtGetMutation gets the binlog mutation for current statement.
 	StmtGetMutation(int64) *binlog.TableMutation
+	// StmtDirtyTablesOP logs the dirty tables operations for current statement.
```

Comment:
`StmtDirtyTablesOP` -> `StmtDirtyTables`.

Reply:
I've change StmtDirtyTables to StmtDirtyTablesOP, because it means an operation.